homeassistant/components/device_tracker/xfinity.py

```diff
@@ -0,0 +1,67 @@
+"""
+Support for Xfinity Gateways.
```

Comment:
We now use a single line docstring: `"""Support for device tracking via Xfinity Gateways."""